task/01-strings-tasks.js

```diff
@@ -22,7 +22,7 @@
  *   '',  'bb'  => 'bb'
  */
 function concatenateStrings(value1, value2) {
-    throw new Error('Not implemented');
+    return value1+value2;
```

Comment:
It would be better to split variable names and operators with spaces to improve readability